src/formatDate.js

```diff
@@ -50,7 +50,35 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldForm = [...fromFormat];
```

Comment:
Use a more descriptive variable name
```suggestion
  const oldFormat = [...fromFormat];
```